app/src/main/java/com/example/sharingang/items/ItemsViewModel.kt

```diff
@@ -139,7 +133,7 @@ class ItemsViewModel @Inject constructor(
             OBSERVABLES.SEARCH_RESULTS -> searchResults
         }
         observable.observe(LifeCycleOwner, {
-            it?.let { adapter.submitList(it) }
+            adapter.submitList(it)
```

Comment:
What happens if "it" is null here? Everything is good or "it" cannot be null or ?

Reply:
"it" is a List<Item> and not a List<Item>?
So Android Studio suggested to remove the check

Reply:
Oh yeah then makes sense, all good!